mesh/mesh.go

```diff
@@ -98,26 +99,39 @@ func (m *Mesh) ValidateLayer(layer *Layer) {
 	}
 }
 
+func SortBlocks(blocks []*Block) []*Block {
+	sort.Slice(blocks, func(i, j int) bool { return blocks[i].ID() < blocks[j].ID() })
+	return blocks
```

Comment:
do you really need to return blocks ? sort doesn't mutate the original slice ?

Reply:
though it quite readable to return the blocks so I get it if you want to keep it.

Reply:
or if you don't mean to change the original array than clone the slice

Reply:
I like returning the value although it's being done in place

Reply:
I see the advantage in readability but we must take care that this will not be confused as returning a completely new different slice/array. as mutating this or the original array will affect both sides.